framework/tlvf/AutoGenerated/src/tlvf/test/tlvVarList.cpp

```diff
@@ -76,6 +76,7 @@ bool tlvTestVarList::alloc_simple_list(size_t count) {
     m_simple_list_idx__ += count;
     *m_simple_list_length += count;
     m_buff_ptr__ += len;
+    if(m_length){ (*m_length) += len; }
```

Comment:
Not relevant to this commit.

Reply:
It is since I changed tlvVarList.yaml to be a TLV class in the previous commit, so it is relevant..